linetools/guis/utils.py

```diff
@@ -212,5 +212,10 @@ def read_spec(ispec, exten=None):
     else:
         raise ValueError('Bad input to read_spec: {:s}'.format(type(ispec)))
 
+    # Normalize?
+    if norm:
+        if spec.co is not None:
```

Comment:
Add `if hasattr(spec, 'co') and...`. I don't think we guarantee the co attribute exists (maybe we should).